app/src/main/java/com/duycuong/weather/utils/TimeUtils.java

```diff
@@ -13,4 +14,17 @@ public static String covertTime(String format, long time) {
                 new SimpleDateFormat(format, Locale.getDefault());
         return simpleDateFormat.format(time * 1000);
     }
+
+    public static long convertDateToTimeStamp(String ngay) {
+        final Date ngayquydoi;
+        long timestamp = 0;
+        SimpleDateFormat sdf = new SimpleDateFormat(Constant.DAY_MONTH);

```

Comment:
WARNING: [SimpleDateFormat] To get local formatting use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()`, or use `new SimpleDateFormat(String template, Locale locale)` with for example `Locale.US` for ASCII dates.
 Almost all callers should use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()` to get a ready-made instance of SimpleDateFormat suitable for the user's locale. The main reason you'd create an instance this class directly is because you need to format/parse a specific machine-readable format, in which case you almost certainly want to explicitly ask for US to ensure that you get ASCII digits (rather than, say, Arabic digits).

Therefore, you should either use the form of the SimpleDateFormat constructor where you pass in an explicit locale, such as Locale.US, or use one of the get instance methods, or suppress this error if really know what you are doing.